README.md

```diff
@@ -1,5 +1,5 @@
 ## A Docker image for the Cantaloupe IIIF image server
-[![Maven Build](https://github.com/ksclarke/freelib-build-tools/workflows/Maven%20Build/badge.svg)](https://github.com/UCLALibrary/docker-cantaloupe/actions) [![Codacy Badge](https://app.codacy.com/project/badge/Grade/74e25a3c2515495aafb6b077fdde9a13)](https://www.codacy.com/gh/UCLALibrary/docker-cantaloupe?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=UCLALibrary/docker-cantaloupe&amp;utm_campaign=Badge_Grade) [![Known Vulnerabilities](https://snyk.io/test/github/uclalibrary/docker-cantaloupe/badge.svg)](https://snyk.io/test/github/uclalibrary/docker-cantaloupe)
```

Comment:
I almost flagged this but I assumed it was fine that the image was coming from that repo as long as the actions URL was correct

Reply:
A copy and paste error :-(